libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -18,11 +23,13 @@ const drillModal = new DrillToModal();
  * If test failed in before each or after each hook DEPARTMENT_ID, PRODUCT_ID could be shifted
  * @returns
  */
-const postDrillDownStepAttributeDF = (value?: string) => {
+
+const postDrillDownStepAttributeDF = (drillFromAttr: string, drillToAttr?: string) => {
```

Comment:
I suggest moving this method to ./tools/api.ts. Then just import it from the api.ts